### PR TITLE
ZENKO-4348 Integration in Zenko is broken since 8.4.20

### PR DESCRIPTION
**Pull request by nicolas2bert**
The following Backbeat commit: [7e11b95d5c7d0701827c63d83d23753dfd5d6119](https://github.com/scality/backbeat/commit/7e11b95d5c7d0701827c63d83d23753dfd5d6119) is introducing an asynchronous call from lifecycle processor to GC through Kafka.

GC entries take a few minutes to get consumed by the GC consumer introducing some flakiness from the archive tests.

Adding a retry seems to help reducing the flakiness.

### CODE REVIEW

**Comment by nicolas2bert** [comment]
ping

File: tests/zenko_tests/node_tests/backbeat/tests/lifecycle/transition.js [hunk 1]
```diff
@@ -88,7 +88,8 @@ const testsToRun = [{
 }];
 
 testsToRun.forEach(test => {
-    describe(`Lifecycle transition from ${test.from} to ${test.to}`, () => {
+    // eslint-disable-next-line prefer-arrow-callback
+    describe(`Lifecycle transition from ${test.from} to ${test.to}`, function () {
```
Comment (naren-rajendran) [inline tests/zenko_tests/node_tests/backbeat/tests/lifecycle/transition.js:92]: why is this needed? the conversion from arrow function!

**Review by naren-rajendran — APPROVED** [review]

File: tests/zenko_tests/node_tests/backbeat/tests/lifecycle/transition.js (hunk 1 above)
Comment (nicolas2bert) [reply tests/zenko_tests/node_tests/backbeat/tests/lifecycle/transition.js:92]: Because of this: https://mochajs.org/#arrow-functions

**Comment by nicolas2bert** [comment]
Before merging:
- Merge https://github.com/scality/backbeat/pull/2331
- Release `8.4.22`
- Update backbeat image tag

File: solution/deps.yaml [hunk 2]
```diff
@@ -6,7 +6,7 @@ backbeat:
   dashboard: backbeat-dashboards
   image: backbeat
   policy: backbeat-policies
-  tag: 8.4.19
+  tag: 8.4.22-pw2
```
Comment (nicolas2bert) [inline solution/deps.yaml]: Will be updated to `8.4.22` before merging.

**Review by alexanderchan-scality — APPROVED** [review]

**Comment by nicolas2bert** [comment]
@bert-e approve
